_episodes_rmd/02-func-R.Rmd

```diff
@@ -49,6 +49,8 @@ The list of argument names are contained within parentheses.
 Next, the [body]({{ page.root }}/reference.html#function-body) of the function--the statements that are executed when it runs--is contained within curly braces (`{}`).
 The statements in the body are indented by two spaces, which makes the code easier to read but does not affect how the code operates.
 
+Aside: the formula here is from [wikipedia](https://en.wikipedia.org/wiki/Fahrenheit#Definition_and_conversion), and defines the zero point for the Kelvin scale as −459.67°F, with a scaling factor of 5/9.
+
```

Comment:
Please merge this aside into the short comment above, as briefly as possible. We don't need to explain the formula itself, because that's not the point of the lesson.

Reply:
@gringer, the formula is, of course, not from Wikipedia. Wikipedia is simply the place that collects information from other sources. I agree with @katrinleinweber in that there is no need to explain the formula in the text.

Reply:
Thanks for d7ee2437e8c3bcf73588b72bf2947fe1324648ff, but still: Is this an added explanation, really needed?

Reply:
Not necessarily, it depends on the audience. The direct conversion from Fahrenheit to Celsius (without any intermediate calculations, as in the current Fahrenheit to Kelvin method) does make function a lot more obvious, so an explanation is less necessary. I think that explanations for formulas are useful to give people a concept that they can hang their understanding on.

Formula both with and without explanation are used in [the previous section](http://swcarpentry.github.io/r-novice-inflammation/01-starting-with-data/index.html). When the conversion from kg to lb is first used, it's just stated in code:
>     # weight in pounds
>     2.2 * weight_kg

Whereas a bit further on (a few steps later) the operations are expanded out:

> In words, we’re asking R to look up the value we tagged weight_kg, multiply it by 2.2, and tag the result with the name weight_lb

Reply:
1. This paragraph switches the discussion from function(s) to _formula_ -- note that text before and after talks about functions, not formula. This is not ideal.
2. If I were picky, I would say that the formula does not _define_ "zero point" (btw, you would have to explain what "zero point" is) -- it defines the rule for converting temperature from one scale to another. Imagine if someone told you that "`w_lb = 2.2 * w_kg` defines zero point for pounds as 0 kg with a scaling factor of 2.2".
3. Lastly, current position of the paragraph makes it appear as this part is essential to understanding how to write functions in R. But it is not, of course. If you feel that it is important to have some explanation -- do that before the discussion of functions begins. This might be helpful/useful: people would first see mathematical expression first and then how wonderful it is in R.  I would suggest doing so in a separate PR, though.

Another note: I'm a Pythonista and can't help but note that the first sentence in the preceding paragraph ("We define... by assigning ... to the output...")  could be a little tiny bit improved. Words define / assign might be new to people. which output?

Yet another note: in the sentence "In this lesson, we'll learn how to write a function...". I'd say "we'll learn how to quickly execute the same set of operations on different data with the help of functions that we can create ourselves." (or something along those lines).

Reply:
> Note that I've changed the explanation in 013b394 to remove the Wikipedia link.

I don't find the link problematic at all, because it can blend into any text. The added "Explanation:" paragraph/line does not.